webpack/ForemanRhCloudFills.js

```diff
@@ -27,6 +29,15 @@ const fills = [
     component: props => <InsightsTotalRiskChartWrapper {...props} />,
     weight: 2800,
   },
+  {
+    slot: 'host-details-page-tabs',
+    name: 'CVEs',
+    component: props => <CVEsHostDetailsTab {...props} />,
+    weight: 300,
+    metadata: {
+      hideTab: vulnrabilityEnabled,
```

Comment:
I think we want the opposite - we want to hide the tab when vulnerability feature is disabled.
```suggestion
      hideTab: vulnrabilityDisabled,
```
or
```suggestion
      hideTab: (hostDetails) =>  { !vulnrabilityEnabled(hostDetails) },
```